ResourceServer/Models/Apartment.cs

```diff
@@ -15,7 +11,10 @@ public class Apartment
         public String ApartmentNumber { get; set; }
         public String ImgThumb { get; set; }
         public String[] ImgList { get; set; }
-        public decimal? Rate { get; set; }
+        public double OwnerRating { get; set; }
+        public double LocationRating { get; set; }
+        public double StandardRating { get; set; }
+        public double PriceRating { get; set; }
```

Comment:
Czemu jest jednak double a nie decimal jak poprzednio?

Reply:
bo w postgresie procedura która zwraca apartamenty automatycznie dzieli przez tą liczbe ocen. Domyślnie wynikiem dzielenia jest double i bez sensu to jeszcze castować na coś innego bo to jest resource-heavy (decimali raczej sie używa przy hajsie gdzie jest mega potrzebna dokładnosć) a te kilka bajtów już nie zrobi różnicy (na floatach nwm czemu nie wysypywała sie procedura)